src/main/java/emissary/core/BaseDataObject.java

```diff
@@ -448,6 +451,9 @@ public int enqueueCurrentForm(final String newForm) {
     public int pushCurrentForm(final String newForm) {
         if (newForm == null) {
             throw new IllegalArgumentException("caller attempted to push a null form value");
+        } else if (newForm.indexOf(KeyManipulator.SEPARATOR) > 0) {
+            // If there is a key separator in the form, then throw an error log as this will cause issues in routing
```

Comment:
Recommend updating language from ```throw an error log``` to ```log an error```. Also consider adding the potential routing issue to the log statement.

With an updated log description, the comment might not be necessary.